department-of-veterans-affairs/caseflow
# Implements video to virtual schedule veteran workflow in new component

Resolves #14894, depends on merging #14938 and #14951 

### Description
Implemented the changes to the ScheduleVeteranForm component to include the virtual hearings state

### Acceptance Criteria

### Testing Plan
1. Run storybook locally: make storybook
1. Navigate to the schedule veteran component Hearings/Components/Schedule Veteran
1. Ensure that all scenarios match the [Figma diagram](https://www.figma.com/proto/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=6022%3A21726&viewport=-269%2C32%2C0.06443598121404648&scaling=min-zoom)
